static/src/stylesheets/module/external/_witness.scss

```diff
@@ -69,54 +69,38 @@
     margin-bottom: 0;
 }
 
-.witness-cta-wrapper {
-    position: relative;
+.witness-cta {
     overflow: hidden;
-    height: 47px;
     margin-bottom: ($gs-baseline/3)*4;
 }
-.witness-cta__body {
-    @include fs-textSans(3);
-    float: left;
-    line-height: 15px;
-    padding: $gs-baseline/2 $gs-gutter/2 0;
-    margin-left: 50px;
 
-    .witness-cta__logo {
-        display: block;
-    }
+.button--witness {
```

Comment:
Also there's this nonsense going on... https://github.com/guardian/frontend/blob/witness-style/static/src/stylesheets/module/external/_witness.scss#L2